articles/install-guide/setup/vs-editors/vs-code.md

```diff
@@ -9,7 +9,21 @@ ms.topic: article
 
 # Installing the Quantum Development Kit for Visual Studio Code #
 
-## Installing Visual Studio Code and the .NET Core SDK (Windows 10, macOS, Linux) ##
+The Quantum Development Kit can be used with Visual Studio Code, Microsoft's open-source cross-platform development environment, making it easy to use Q# with C# or Python. Additionally, Visual Studio Code is available for Windows, Linux, and macOS.
+
+Below, you will first find instructions for installing the Quantum Development Kit extension directly--follow these if you already have Visual Studio Code.
+If you do not, then you may skip ahead to the "Installing Visual Studio Code" section.
+
+Lastly, we take you through running a sample application to validate that your Q# environment has been correctly installed and configured. 
+Upon successful validation, you will be ready to start developing!
+
+## Already have Visual Studio Code and the .NET Core SDK? ##
```

Comment:
Same comment as for VS 2019:
I think it might be more concise to simplify this section into:
Installing Visual Studio Code (I think people will naturally skip/skim this if they already have it installed)
Installing the extension
Validating the install